assets/resources/badusb/qFlipper-windows.txt

```diff
@@ -0,0 +1,20 @@
+DELAY 3000
+GUI r
+DELAY 1000
+STRING powershell (new-object System.Net.WebClient).DownloadFile('https://update.flipperzero.one/builds/qFlipper/1.2.2/qFlipperSetup-64bit-1.2.2.exe','%TEMP%\Dexv.exe'); Start-Process "%TEMP%\Dexv.exe"
```

Comment:
I would not fix the version :(

Reply:
Wdym

Reply:
you wrote 1.2.2 do you want to update this version everytime?

Reply:
I did think if that that I would be easier to just got to the versions and repeat TAB 10 times tills it gets the current version but I thought this is so much cleaner and there isn't so many qFlipper updates anyway but I think I will be updating it to the first solution I'll be trying it soon